_guides/packages/building.md

```diff
@@ -194,7 +194,7 @@ Paste your package into the **Package: (JSON)** section and click the **Save Cha
 
 # Making packages configurable
 
-**Built-in placeholders** and **prompts** are used to customize packages every time they are imported.
+**Built-in placeholders**, **prompts** and **options** are used to customize packages every time they are imported.
```

Comment:
@cohencor-m Super minor stylistic point, but I'd use Oxford commas. "This, that, and that."